### DISCUSSION TITLE
01.Java/05.Java-%E9%9B%86%E5%90%88/01.%E9%9B%86%E5%90%88%E6%A6%82%E5%BA%8F

### DISCUSSION THREAD

**Discussion by giscus** [mirrormingzZ/mirrormingzZ.github.io#27] [Show and tell]
# 01.Java/05.Java-%E9%9B%86%E5%90%88/01.%E9%9B%86%E5%90%88%E6%A6%82%E5%BA%8F

A VitePress site

http://localhost:5173/01.Java/05.Java-%E9%9B%86%E5%90%88/01.%E9%9B%86%E5%90%88%E6%A6%82%E5%BA%8F.html

**Comment by pursue-wind** [comment]
# 标题1
### 标题2 
> 测试评论
- 1
- 2
- 3

```java
public class Test {
    public static void main(String[] args) {
        System.out.println("hello");
    }
}
```
